lib/rbac/access.rb

```diff
@@ -25,22 +25,40 @@ def accessible?
     end
 
     def id_list
-      ids = collect_ids
-      Rails.logger.info("IDS for #{@app_name} #{ids}")
-      ids.include?('*') ? [] : ids
+      ids
```

Comment:
Possibly rename this to `generate_ids`? The current name was making it hard to understand why `ids` had to be called and then iterating over the `@ids` instance variable.  
I think it is easier to understand with something like below.

```
def id_list 
  generate_ids
  @ids.include?...
```

Reply:
@syncrou Done, please review